### PR TITLE
feat: add `splash-cli` spec

### PR DESCRIPTION
**Pull request by Rawnly**
Added [splash-cli](https://github.com/splash-cli/splash-cli) spec.

### CODE REVIEW

File: dev/splash.ts [hunk 1]
```diff
@@ -0,0 +1,314 @@
+const completionSpec: Fig.Spec = {
+  name: "splash-cli",
+  displayName: "Splash",
+  description: "Get stunning wallpapers from Unsplash",
+  subcommands: [
+    {
+      priority: 100,
```
Comment (fedeci) [inline dev/splash.ts:7]: Having fixed priorities for each subcommand does not allow fig to give priority to the most used commands, I would remove all of them when not necessary.

File: dev/splash.ts [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+          name: "get",
+          args: {
+            name: "config key",
+            description: "Config key",
+            isOptional: true,
+          },
+        },
+        {
+          name: "set",
+        },
```
Comment (fedeci) [inline dev/splash.ts:21]: Can you add a description for this. I also think that it accepts an arg.

File: dev/splash.ts (hunk 1 above)
Comment (Rawnly) [reply dev/splash.ts]: Oh ok! Fixed in [`f4b6b8b` (#547)](https://github.com/withfig/autocomplete/pull/547/commits/f4b6b8b5615d83dcf6ede3fc421ee191119f25eb)

File: dev/splash.ts [hunk 3]
```diff
@@ -0,0 +1,309 @@
...
+            {
+              name: "key",
+            },
+            {
+              name: "value",
+            },
+          ],
+        },
+        {
+          name: "help",
```
Comment (fedeci) [inline dev/splash.ts:52]:
```suggestion
          name: "help",
          description: "Show help menu",
```

File: dev/splash.ts [hunk 4]
```diff
@@ -0,0 +1,309 @@
...
+    },
+    {
+      name: "user",
+      subcommands: [
+        {
+          name: "login",
+        },
+        {
+          name: "logout",
+        },
```
Comment (fedeci) [inline dev/splash.ts:136]: Same here

File: yarn.lock [hunk 5]
```diff
@@ -0,0 +1,3059 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (fedeci) [inline yarn.lock:1]: Can you remove this from the PR? We are using npm😄

File: dev/splash.ts (hunk 2 above)
Comment (Rawnly) [reply dev/splash.ts:21]: It doesn't! (I'm the author of the cli 😉)

Comment (fedeci) [reply dev/splash.ts]: I've seen that later😅

**Pull request merged**